A3A/addons/GUI/CfgFunctions.hpp

```diff
@@ -28,5 +28,9 @@ class CfgFunctions {
             class recruitSquadDialog {};
             class requestMissionDialog {};
         };
+        class ObjectHelpers {
+            file = QPATHTOFOLDER(functions\ObjectHelpers);
+            class GUI_sizeOf {};
+        };
     };
```

Comment:
i would like if we move the new helper function to a new base class using the ADDON macro (resolves to "A3A_GUI" in this case) and getting rid of prefixing in the file name.
```suggestion
    };

    class ADDON {
        class ObjectHelpers {
            file = QPATHTOFOLDER(functions\ObjectHelpers);
            class sizeOf {};
        };
    };
```

Reply:
may I ask what the purpose of this would be, as every macro comes with the cost of somewhat obfuscating the code and raising complexcity. So this should be balanced.

Do we plan on ever changing the prefix or is there a plan to make the prefixes dynamic?

Reply:
Boilerplate for thimgs like the use of the ADDON macro, it will also help developers know where something is from, reduces manually typing out prefixes in internal addon function calls etc.

also while obfuscation would be problematic in any other languages these macros are community standard among arma 3 mod developers, hence why I am pushing a bit for their use as any sqf developer would likely be familiar if they have worked on mods before and not just missions.

Probably some more stuff I can't remember rn tbh

Reply:
Did not know that, if it is somewhat standard then I have no problem with it. I was just hoping to be able to use the functions.hpp to get a functions and prefix list via static analysis.